src/python_testing/TC_ACE_1_5.py

```diff
@@ -91,6 +77,9 @@ async def test_TC_ACE_1_5(self):
         self.th2 = new_fabric_admin.NewController(nodeId=TH2_nodeid,
                                                   paaTrustStorePath=str(self.matter_test_config.paa_trust_store_path))
 
+        params = self.openCommissioningWindow(self.th1, TH1_nodeid, self.matter_test_config.discriminators[0])
```

Comment:
This should be the DUT node id - this is attempting to open a commissioning window on itself.

Reply:
yes you right, _my bad_. I fixed